raft/server.cc

```diff
@@ -1452,6 +1452,9 @@ void server_impl::register_metrics() {
 
         sm::make_gauge("in_memory_log_size", [this] { return _fsm->in_memory_log_size(); },
              sm::description("size of in-memory part of the log"), {server_id_label(_id)}),
+
+        sm::make_gauge("status", [this] { return _aborted ? 1 : 0; },
+                       sm::description("the status of the raft group server, 0 - normal, 1 - aborted"), {server_id_label(_id)}),
```

Comment:
I think (and we discussed it on the call) the metric should be on Scylla side and have to have three values: 0 - disabled, 1 - normal, 2 - aborted. And it should have a group0 specific name. Otherwise a management will have a hard time to know which `server_id` is group0 one.